io.github.chilledheart.yass.yml

```diff
@@ -0,0 +1,33 @@
+id: io.github.chilledheart.yass
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: yass
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=~/.yass
```

Comment:
```suggestion
  - --persist=.yass
```

Reply:
It is due to some legacy issues that we use the same configuration directory for both gui client and command line client.

And after some tests, "--persist" doesn't work as expected.

Reply:
> And after some tests, "--persist" doesn't work as expected.

What does not work?

Reply:
yass stores all configuration under `~/.yass` directory.

With `--persist=~/.yass`, it no longer reads or writes its configuration. I tested with:
1. run `flatpak run --user io.github.chilledheart.yass` and change the default values.
2. kill current instance
3. re-run `flatpak run --user io.github.chilledheart.yass`
4. previous changes don't present.

With `--filesystem=~/.yass`, it works.

Reply:
`--persist` cannot be passed as an argument to `flatpak run`, also it is `--persist=.yass` not `--persist=~/.yass`

Reply:
Cool. done in 4a7a30f.